app/schemas/user.py

```
@@ -0,0 +1,6 @@
+from pydantic import BaseModel, EmailStr
```

Ruff (0.8.2)

`pydantic.EmailStr` imported but unused

Remove unused import: `pydantic.EmailStr`

(F401)